databricks_cli/pipelines/cli.py

```diff
@@ -49,28 +50,147 @@
 
 
 @click.command(context_settings=CONTEXT_SETTINGS,
-               short_help='Deploys a pipeline according to the pipeline specification.')
-@click.argument('spec_arg', default=None, required=False)
-@click.option('--spec', default=None, type=PipelineSpecClickType(), help=PipelineSpecClickType.help)
+               short_help='Creates a pipeline according to the pipeline settings.')
+@click.argument('settings_arg', default=None, required=False)
+@click.option('--settings', default=None,
+              type=PipelineSettingClickType(), help=PipelineSettingClickType.help)
 @click.option('--allow-duplicate-names', is_flag=True,
-              help="If true, skips duplicate name checking while deploying the pipeline.")
+              help="If true, skips duplicate name checking while creating the pipeline.")
+@debug_option
+@profile_option
+@pipelines_exception_eater
+@provide_api_client
+def create_cli(api_client, settings_arg, settings, allow_duplicate_names):
+    """
+    Creates a pipeline according to the pipeline settings. The pipeline settings are a
```

Comment:
Suggested change:

"Creates a pipeline. Pipeline settings are defined in a JSON document. The JSON settings document can be passed directly to the command or stored in a file. To use a file containing the pipeline settings, use the --settings argument to pass the file path to the command. On success, the URL and ID of the new pipeline are logged to STDOUT."